mobx_lint/pubspec.yaml

```diff
@@ -0,0 +1,21 @@
+name: mobx_lint
```

Comment:
for publishing this package, we need to make changes to the github workflow as well...pls see

Reply:
@pavanpodila done